client/src/components/ShoeShowPage.js

```diff
@@ -14,30 +16,32 @@ const ShoeShowPage = props => {
               }
             const shoeData = await response.json()
             setShoe(shoeData.shoe)
+            setReviews(shoeData.shoe.reviews)
         }   catch(err) {
             console.error(`Error in fetch: ${err.message}`)
         }
     }
-
-
+    
     useEffect(() => {
         getShoe()
     }, [])
 
-
     return(
-        <div className = "callout">
-            <h1>{shoe.name}</h1>
-            <img src={shoe.url} />
-            <div>
-                <h5>Category: {shoe.category}</h5>
-                <h5>Color: {shoe.color}</h5>
-                <h6>Description:</h6>
-                <p>
-                    {shoe.description}
-                </p>
-            </div>
-        </div>
+          <div className = "callout">
+              <h1>{shoe.name}</h1>
+              <img src={shoe.url} />
+              <div>
+                  <h5>Category: {shoe.category}</h5>
+                  <h5>Color: {shoe.color}</h5>
+                  <h6>Description:</h6>
+                  <p>
+                      {shoe.description}
+                  </p>
+                  <div>
+                    <ReviewList reviews={reviews}/>
+                  </div>
```

Comment:
the ReviewList component itself returns a top-level `div` element with the ReviewTiles - this `div` wrapping around the ReviewList should be removed to clear up the DOM elements in the HTML